app/src/ui/preferences/appearance.tsx

```diff
@@ -118,15 +132,41 @@ export class Appearance extends React.Component<
     return (
       <DialogContent>
         <h2 id="theme-heading">Theme</h2>
-
-        <RadioGroup<ApplicationTheme>
-          ariaLabelledBy="theme-heading"
-          className="theme-selector"
-          selectedKey={selectedTheme}
-          radioButtonKeys={themes}
-          onSelectionChanged={this.onSelectedThemeChanged}
-          renderRadioButtonLabelContents={this.renderThemeSwatch}
-        />
+        <Row>
+          <RadioGroup<ApplicationTheme>
+            ariaLabelledBy="theme-heading"
+            className="theme-selector"
+            selectedKey={selectedTheme}
+            radioButtonKeys={themes}
+            onSelectionChanged={this.onSelectedThemeChanged}
+            renderRadioButtonLabelContents={this.renderThemeSwatch}
+          />
+        </Row>
+
+        <div className="advanced-section">
```

Comment:
To make clear that this is scoped to a specific platform, could we extract this into it's own function?

Rough patch:

```diff
diff --git a/app/src/ui/preferences/appearance.tsx b/app/src/ui/preferences/appearance.tsx
index f7b15caf2..83c8b9aa8 100644
--- a/app/src/ui/preferences/appearance.tsx
+++ b/app/src/ui/preferences/appearance.tsx
@@ -112,6 +112,39 @@ export class Appearance extends React.Component<
     }
   }
 
+  private renderTitleBarStyle() {
+    if (!__LINUX__) {
+      return null
+    }
+
+    return (
+      <div className="advanced-section">
+        <h2>Title bar style</h2>
+
+        <Select
+          value={this.state.titleBarStyle}
+          onChange={this.onSelectChanged}
+        >
+          <option value="native">Native</option>
+          <option value="custom">Custom</option>
+        </Select>
+
+        <div className="git-settings-description">
+          <ul>
+            <li>
+              <b>Native</b>: Uses the menu system and chrome provided by your
+              window manager.
+            </li>
+            <li>
+              <b>Custom</b>: Uses the menu system provided by GitHub Desktop,
+              hiding the default chrome provided by your window manager.
+            </li>
+          </ul>
+        </div>
+      </div>
+    )
+  }
+
   public render() {
     const { selectedTheme } = this.state
 
@@ -143,30 +176,7 @@ export class Appearance extends React.Component<
           />
         </Row>
 
-        <div className="advanced-section">
-          <h2>Title bar style</h2>
-
-          <Select
-            value={this.state.titleBarStyle}
-            onChange={this.onSelectChanged}
-          >
-            <option value="native">Native</option>
-            <option value="custom">Custom</option>
-          </Select>
-
-          <div className="git-settings-description">
-            <ul>
-              <li>
-                <b>Native</b>: Uses the menu system and chrome provided by your
-                window manager.
-              </li>
-              <li>
-                <b>Custom</b>: Uses the menu system provided by GitHub Desktop,
-                hiding the default chrome provided by your window manager.
-              </li>
-            </ul>
-          </div>
-        </div>
+        {this.renderTitleBarStyle()}
       </DialogContent>
     )
   }
```

Reply:
Will this be clearer?

```jsx
{__LINUX__ && this.renderTitleBarStyle()}
```

Reply:
OK. Waiting for conclusion on https://github.com/shiftkey/desktop/pull/912#discussion_r1285218484 to proceed (they touch the same piece of code).

Reply:
Reply'd https://github.com/shiftkey/desktop/pull/912#discussion_r1306449445